config/routes.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+  namespace :api do
+    get 'random_greeting', to: 'messages#random_greeting'
+  end
+end
```

Comment:
- [ ] As required for the project and as mentioned the given **[tutorial](https://betterprogramming.pub/react-with-rails-2022-bd28e1fcd355#ad0a)**, kindly ensure that there is a main static route created and a controller with one action.
![Screenshot from 2023-07-20 03-32-10](https://github.com/OsvaldoBC/hello-rails-react/assets/83042742/4f64b93c-7cff-4f18-8ec2-78663c89baff)